lib/container_orchestrator.rb

```diff
@@ -0,0 +1,29 @@
+
```

Comment:
Kill this blank line please.  (You vim users and your leading blank lines :trollface: )